### PR TITLE
Add tooltip message for `Get TFT` button 

### PR DESCRIPTION
**Pull request by AlaaElattar**
- https://github.com/threefoldtech/tfgrid-sdk-ts/issues/1143
- This message shows only with qa && main networks.

Screenshot from 2023-10-15 12-57-00

Screenshot from 2023-10-15 12-57-05

### CODE REVIEW

File: packages/playground/src/components/funds_card.vue [hunk 1]
```diff
@@ -1,15 +1,24 @@
...
+        class="px-lg-6 px-md-2 px-sm-0 mx-sm-0"
+        style="color: white; max-width: 140px; width: auto; background-color: var(--primary)"
+        :loading="loadingAddTFT"
+        v-bind="props"
+      >
+        GET TFT
+      </v-btn>
+    </template>
+    <div>
+      Get TFT using BTC or Credit card to Stellar bridge, then use bridge to get your TFT from Stellar.
```
Comment (xmonader) [inline packages/playground/src/components/funds_card.vue:18]: Get TFT on Stellar using BTC or Credit card, then you can bridge it to your tfchain account using the Bridge in the dashboard

File: packages/playground/src/components/funds_card.vue [hunk 2]
```diff
@@ -1,15 +1,25 @@
...
-    width="2000"
-    color="white"
-    @click="addTFT"
-    class="px-lg-6 px-md-2 px-sm-0 mx-sm-0"
-    style="color: white; max-width: 140px; width: auto; background-color: var(--primary)"
-    :loading="loadingAddTFT"
-  >
-    GET TFT
-  </v-btn>
+  <v-tooltip location="bottom" v-if="showTooltip()" close-delay="1000" color="primary">
```
Comment (MohamedElmdary) [inline packages/playground/src/components/funds_card.vue:2]: Do we want to remove `GET TFT` button in main/test net?

Comment (AlaaElattar) [reply packages/playground/src/components/funds_card.vue:2]: no, we want to add tooltip to button only at qa && main net.

Comment (MohamedElmdary) [reply packages/playground/src/components/funds_card.vue:2]: @AlaaElattar Then you can disabled the tooltip not removing it
Check docs https://vuetifyjs.com/en/api/v-tooltip/#props-disabled

**Comment by xmonader** [comment]
Let's have the tooltip regardless of the network

**Comment by AlaaElattar** [comment]
- replace `More Information` with info icon that navigates to manual for more details

image

**Review by maayarosama — APPROVED** [review]

**Review by MohamedElmdary — APPROVED** [review]

**Pull request merged**